src/components/Home.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';

```

Comment:
- [ ] Kindly stick with the design guidelines(There are two colors for design one red and one blue) for the `home` page for `colors`, `background-colors`, and `alignment`.

| Your design | Design guidelines |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/170547718-c49a7f57-8915-4e1e-a536-215efa3422d4.png) | ![image](https://user-images.githubusercontent.com/24703171/170547504-11030085-b9bd-4f80-ae94-bb0e464ed108.png) |